elements/h1-h6.html

```diff
@@ -0,0 +1,50 @@
+<article class="article">
+  <header class="article-header">
+    <h2>
+      <code>
+        &lt;h1-h6&gt;
+      </code>
+    </h2>
+    <div class="base-words">
+      <div class="base-word-line">
+        English Base: <strong>Header</strong>
+      </div>
+      <div class="base-word-line">
+        Spanish translation/Traducción en Español: <strong></strong>
+      </div>
+    </div>
+  </header>
+  <div class="definition">
+    <div class="definition-box definition-spanish">
+      <h4>Definici&oacute;n</h4>
+      <p></p>
+    </div>
+    <div class="definition-box definition-english">
+      <h4>Definition</h4>
+      <p>Not to be confused with the &lt;header&gt; element which is a whole section of a web page: heading elements are used to describe the topic or give the title of the section it is introducing.  There are six graduated levels of font size and boldness with &lt;h1&gt; being the largest and boldest down to &lt;h6&gt; as the smallest and least boldface.</p>
+    </div>
+  </div>
+  <div class="how-to-remember">
+    <div class="how-to-remember-box how-to-remember-spanish">
+      <h4>&iquest;C&oacute;mo recordar?</h4>
+      <p></p>
+    </div>
+    <div class="how-to-remember-box how-to-remember-english">
+      <h4>How to Remember</h4>
+      <p>In a concert, the number one artist headlining in the show is the biggest and brightest, the second in importance is slightly smaller and under the headliner, and so on down the line - just as heading level 1 &lt;h1&gt; is the biggest and boldest, heading level 2 &lt;h2&gt; is slightly smaller and less bold, and so on down the line.</p>
```

Comment:
this is a run on sentence.
you could try something like this:
"In a concert, the number one artist headlining in the show is the biggest and brightest. The second in importance is slightly smaller and under the headliner, and so on down the line. Just as heading level 1 <h1> is the biggest and boldest, heading level 2 <h2> is slightly smaller and less bold."